### DISCUSSION TITLE
Django-salesman tutorial

### DISCUSSION THREAD

**Discussion by zafon18** [dinoperovic/django-salesman#14] [General]
Django-salesman tutorial

Hey @dinoperovic,
I would like to thank you for the project you have made and ask for help :)
Is there a tutorial on how to make a simple shop using your library?
Do you need to use a headless solution in your library or can you use the regular django views?
Thanks for the answer

**Comment by dinoperovic** [comment]
Hi @zafon18, this library is focused on providing the shop API intended for headless use.

However, nothing is stoping you from implementing your own views. You would just have to re-write some logic using django forms rather than current serializers to get it to work.

This is not supported in the package by default to make it more maintainable for myself, as I don't have much time to do so.
